pkg/internal/mysqlcluster/defaults_test.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2020 Pressinfra SRL
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mysqlcluster
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+var _ = Describe("MySQL defaults", func() {
+	It("humanize should not round the value", func() {
+		q := resource.MustParse("1.5Gi")
+		hq := humanizeSize(q.Value())
+		Expect(hq.String()).To(Equal("1.5G"))
```

Comment:
Does MySQL work with `1.5G` or you need to pass `1500M`?

Reply:
I've updated `humanizeSize` as follows:
 * it outputs values only in megabytes (`M`);
 * for values less than 1M ti will use bytes.